tests/prism/swagger.json

```diff
@@ -88,7 +97,42 @@
           "200": {
             "description": "OK",
             "schema": {
-              "$ref": "#/definitions/VerificationResponse"
+              "type": "object",
+              "properties": {
+                "result": {
+                  "type": "object",
+                  "properties": {
+                    "page": {
+                      "type": "integer",
+                      "minimum": 0
+                    },
+                    "total_pages": {
+                      "type": "integer",
+                      "minimum": 0
+                    },
+                    "per_page": {
+                      "type": "integer",
+                      "minimum": 0
+                    },
+                    "total_items": {
+                      "type": "integer",
+                      "minimum": 0
+                    },
+                    "items": {
+                      "type": "array",
+                      "items": {
+                        "type": "object",
+                        "title": "Certificate",
```

Comment:
This json will need to be updated since the titles are mismatched (please refer to our "internal" core-api PR where I point them out)

Reply:
Yep! Done, thanks :)